bin/katello-certs-check

```diff
@@ -161,8 +161,8 @@ function check-ca-bundle () {
 }
 
 function check-ca-bundle-size () {
-    printf "Checking CA bundle size: "
     CHECK=$(grep -c "^--*BEGIN" $CA_BUNDLE_FILE)
+    printf "Checking CA bundle size: $CHECK"
```

Comment:
If there's anything on STDERR that will now be printed before the `Checking ...` part is printed. I think `printf` doesn't have a line ending so it might be better to call `printf "$CHECK"` separate.

Reply:
Fair point, updated